src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String INVALID_COMMAND_PRINTER = "One hour of lifespan has been deducted, in accordance with our Terms and Services.";
+    private static final String VALID_COMMAND_PRINTER = "Command acknowledged. Reducing user lifespan by 30 minutes.";
+    private static final String LOGO =
+              "    // | |     //   ) )  // | |  \\\\ / / \\\\    / / //   ) ) \n"
+            + "   //__| |    //___/ /  //__| |   \\  /   \\\\  / / ((        \n"
+            + "  / ___  |   / ___ (   / ___  |   / /     \\\\/ /    \\\\      \n"
+            + " //    | |  //   | |  //    | |  / /\\\\     / /       ) )   \n"
+            + "//     | | //    | | //     | | / /  \\\\   / / ((___ / /    ";
+    private static String command;
+    private static ArrayList<Task>tasks = new ArrayList<>();
+    private static Scanner in = new Scanner(System.in);
+    private enum taskType{
+        TODO,DEADLINE,EVENT,INVALID
+    }
+    private static void printList(){
+        System.out.println(LINE);
+        System.out.println(VALID_COMMAND_PRINTER);
+        for(int i=0;i<tasks.size();i++){
+            System.out.print(i+1+":");
+            System.out.println(tasks.get(i).toString());
+        }
+        System.out.println(LINE);
+    }
+    private static void mark(String[] commands){
```

Comment:
I would like if there are JavaDoc styled comments on your methods so I am able to understand your code better.